MeshLib/MeshEditing/FlipElements.cpp

```diff
@@ -56,7 +56,7 @@ std::unique_ptr<MeshLib::Mesh> createFlippedMesh(MeshLib::Mesh const& mesh)
     for (std::size_t i=0; i<n_elems; ++i)
         new_elems.push_back(createFlippedElement(*elems[i], new_nodes).release());
 
-    MeshLib::Properties new_props (mesh.getProperties());
+    const MeshLib::Properties& new_props (mesh.getProperties());
```

Comment:
A new mesh is generated here. Won't it couse memory issues if you pass a reference here, i.e. both meshes point to the same property vector?